SSD/SSDDevice.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+class SSD {
```

Comment:
class 이름 을 변경하는게 좋아 보입니다.
```suggestion
class SSDDevice {
```